### PR TITLE
Bug Prediction #2

### PR DESCRIPTION
**Pull request by fletna**
Elaborated on motivation, research questions, added citations.

### CODE REVIEW

**Comment by Inventitech** [comment]
This is considerably behind what other groups have done so far. Have a look at other PRs.

I would merge this once you rename your file to `04-bug-prediction.Rmd` and rebase to the current master.

**Comment by Inventitech** [comment]
Conflicts in bib

File: bug_prediction.Rmd [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* @Gyimothy2005
+* @Catal2009review
+* @Arisholm2010
+* @DAmbros2010
+* @Hall2012
+* @Lewis2013
+
+Our searches were based on the following elements:
+
+1. Keyword search using search engines (Scopus, ACM Digital Library, IEEE Explorer).
```
[the remark this answers is not in the document]
Comment (chadepl) [reply bug_prediction.Rmd]: I agree! But maybe you could elaborate a bit more on why selecting papers after 2008 and with more than 10 citations? Especially in the first case, it is not clear to me the rationale behind the decision.

**Comment by Inventitech** [comment]
I'll merge this for now, but this is far below even the minimal requirements to pass the course. You need to gain speed, fast.

**Comment by Inventitech** [comment]
One thing I want to see addressed in the next PR is to really answer your RQs with the help of the identified literature.

**Comment by mpsijm** [comment]
Aha, so this is the PR that duplicated our BibTeX references in book.bib... :sweat_smile: Look in the diff for `cesar2017a`, which is our first listed paper. All papers below that until `adams2016a` are duplicated :sweat_smile:

**Comment by Inventitech** [comment]
> @mpsijm yes unsure what happened there

You haven't really merged -- the merge markers are still in there (=======).

**Comment by fletna** [comment]
@Inventitech so what's the best way of fixing this for everyone?

**Comment by Inventitech** [comment]
Well, it's in master now, so just checkout master, correct the bib and file a new PR with just the corrections. I'll promptly merge this.

**Pull request merged**